core/src/Cuda/Kokkos_CudaSpace.cpp

```diff
@@ -136,6 +136,22 @@ void CudaSpace::access_error() {
   Kokkos::Impl::throw_runtime_exception(msg);
 }
 
+#ifdef KOKKOS_IMPL_DEBUG_CUDA_SERIAL_EXECUTION
+void CudaSpace::cuda_set_serial_execution(bool val) {
+  CudaSpace::kokkos_impl_cuda_use_serial_execution_v = val;
+}
+bool &CudaSpace::cuda_use_serial_execution() {
+  return CudaSpace::kokkos_impl_cuda_use_serial_execution_v;
+}
+
+extern "C" void kokkos_impl_cuda_set_serial_execution(bool val) {
```

Comment:
@dalg24 this is a way of avoiding an extern variable like we use in the other debug PR. Is this idiom okay?